react/tic-tac-toe/src/redux/store.js

```diff
@@ -1,13 +1,20 @@
 import { createStore, compose, combineReducers, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
+import { reducer as formReducer } from 'redux-form';
+import { connectRouter, routerMiddleware } from 'connected-react-router';
+import { createBrowserHistory } from 'history';
 
 import game from './game/reducer';
 import matches from './matches/reducer';
 
+export const history = createBrowserHistory();
+
 const reducer = combineReducers({
   game,
-  matches
+  matches,
+  form: formReducer,
```

Comment:
you can import reducer as form so you can put `form` here and continue the format.